internal/kafka/command_cluster_describe.go

```diff
@@ -32,6 +32,7 @@ type describeStruct struct {
 	ServiceProvider    string `human:"Provider" serialized:"provider"`
 	Region             string `human:"Region" serialized:"region"`
 	Availability       string `human:"Availability" serialized:"availability"`
+	Network            string `human:"Network,omitempty" serialized:"network,omitempty"`
```

Comment:
When would the network be empty? Shouldn't a cluster always have an associated network?

Reply:
If my understanding is correct, we only want to display networks for dedicated clusters with private connectivity. It'd be empty for basic/standard and public dedicated clusters; these public clusters don't return a network anyhow. CC @kabraham13

Reply:
And Enterprise clusters.. as they are not 'in' a network either (a customer facing one that is)